src/GitHook/Command/FileCommand/PreCommit/PhpMdCheckCommand.php

```diff
@@ -70,7 +70,7 @@ public function run(CommandContextInterface $context): CommandResultInterface
         if (!$process->isSuccessful()) {
             $commandResult
                 ->setError(trim($process->getErrorOutput()))
-                ->setMessage(trim($process->getOutput()));
+                ->setMessage(trim($process->getOutput() ?: $process->getErrorOutput()));
```

Comment:
Why is this needed? `ErrorOutput` is already in the `Error`.

Reply:
$process->getOutput() может выдавать пустую строку, и не понятно где ошибка возникла. Это я воспроизвел, когда path to file не валидный

Reply:
I still get the same error message when I revert this line. Are you sure this is useful?

Reply:
<img width="991" alt="Screenshot 2020-03-30 at 13 10 23" src="https://user-images.githubusercontent.com/15310986/77901169-d224cd00-7287-11ea-9307-4d40ee4624ad.png">

Now it's OK because you see exception from PhpMdCheckConfiguration class. But when we have exception in vendor/bin/phpmd we have empty red line. With this fix we have readable message that ruleset file not found